src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/DeserializationHelpers.cs

```diff
@@ -6,9 +6,13 @@
 using System.Collections.Generic;
 using System.Diagnostics.Contracts;
 using System.Globalization;
+using System.Net.Http;
 using System.Reflection;
 using System.Runtime.Serialization;
+using Microsoft.AspNet.OData.Adapters;
 using Microsoft.AspNet.OData.Common;
+using Microsoft.AspNet.OData.Extensions;
+using Microsoft.Extensions.DependencyInjection;
```

Comment:
Looks like this import is not necessary.

Reply:
They are both necessary. One works for AspNet classic, the other for AspNet Core